sklearn/kernel_ridge/kernel_ridge.py

```diff
@@ -81,27 +63,25 @@ class KernelRidge(BaseEstimator, RegressorMixin):
       "Machine Learning: A Probabilistic Perspective", The MIT Press
       chapter 14.4.3, pp. 492-493
 
+    * An, Senjian, Wanquan Liu, and Svetha Venkatesh.
+      "Face recognition using kernel ridge regression." Computer Vision
+      and Pattern Recognition, 2007. CVPR'07. IEEE Conference on. IEEE,
+      2007.
+
     See also
     --------
     sklearn.linear_model.Ridge:
         Linear ridge regression.
     sklearn.svm.SVR:
         Support Vector Regression implemented using libsvm.
+    sklearn.svm.SVC:
+        Support Vector Classification implemented using libsvm.
 
-    Examples
-    --------
-    >>> from sklearn.kernel_ridge import KernelRidge
-    >>> import numpy as np
-    >>> n_samples, n_features = 10, 5
-    >>> rng = np.random.RandomState(0)
-    >>> y = rng.randn(n_samples)
-    >>> X = rng.randn(n_samples, n_features)
-    >>> clf = KernelRidge(alpha=1.0)
-    >>> clf.fit(X, y) # doctest: +NORMALIZE_WHITESPACE
-    KernelRidge(alpha=1.0, coef0=1, degree=3, gamma=None, kernel='linear',
-                kernel_params=None)
     """
-    def __init__(self, alpha=1, kernel="linear", gamma=None, degree=3, coef0=1,
+    # By default, it works as a regressor
+    regression = True
```

Comment:
I don't think we need a default. Just put this in the regressor (or don't, which is my preference)

Reply:
Replied in your review about `_estimator_type`